samples/DurableFunctions/LocalFunctionProj/LocalFunctionProj.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>

```

Comment:
non-blocking: any reason not to choose .net8, especially given the M1/M2 issues we've encountered w/ .NET7?

Reply:
yeah, I kind of mentioned that in the description of the PR. The reason was the original guidance [here](https://github.com/Azure/azure-functions-on-container-apps/tree/main?tab=readme-ov-file#create-the-local-function-project-c-isolated---http-trigger) uses a .NET 7 isolated image. I wasn't sure if the .NET 8 isolated image is ready. To be on the safe side, I chose what's proven to work already. That's why I put testing .NET 8 as the next step.

Reply:
(I'll try to do it today)

Reply:
I wouldn't stress about it, .NET 7 is fine :) . Just wondering. If it's easy, let's do it. If it's not, then it's fine.

Reply:
.Net 8 is supported as well. Readme page is updated